decouple.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+
+import collections
+import fnmatch
+import json
+import pathlib
+import subprocess
+import sys
+
+import yaml
+
+CURRENT_DIR = pathlib.Path(__file__).parent.absolute()
+
+
+def generate_dependency_graph(*args):
+    argstring = ' '.join(args)
+    command = f'pydeps --nodot --show-deps {argstring}'
```

Comment:
```suggestion
    command = shlex.join(["pydeps", "--nodot", "--show-deps", *args])
```

Reply:
Since we don't use shell=True now new can just pass the args.